addons/mail/static/src/components/thread_viewer/thread_viewer_tests.js

```diff
@@ -320,6 +320,7 @@ QUnit.test('mark channel as fetched when a new message is loaded and as seen whe
         }
     });
     const thread = this.env.models['mail.thread'].create({
+        composer: [['create']], // avoid initial focus
```

Comment:
Er... That's off, and that's because we shouldn't force focus composer on thread creation.
Even with the auto-focus feature by default, this workaround is not intuitive at all.